test/pools/ipc_jemalloc_devdax.cpp

```diff
@@ -0,0 +1,25 @@
+// Copyright (C) 2024 Intel Corporation
+// Under the Apache License v2.0 with LLVM Exceptions. See LICENSE.TXT.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include "umf/pools/pool_jemalloc.h"
+#include "umf/providers/provider_devdax_memory.h"
+
+#include "ipcFixtures.hpp"
+
+HostMemoryAccessor hostAccessor;
+
+auto defaultDevDaxParams = umfDevDaxMemoryProviderParamsDefault(
+    getenv("UMF_TESTS_DEVDAX_PATH"),
+    atol(getenv("UMF_TESTS_DEVDAX_SIZE") ? getenv("UMF_TESTS_DEVDAX_SIZE")
+                                         : "0"));
+
+static std::vector<ipcTestParams> ipcJemallocPoolTestParamsList = {
+    {umfJemallocPoolOps(), nullptr, umfDevDaxMemoryProviderOps(),
+     &defaultDevDaxParams, &hostAccessor, false},
+};
+
+GTEST_ALLOW_UNINSTANTIATED_PARAMETERIZED_TEST(umfIpcTest);
+
+INSTANTIATE_TEST_SUITE_P(DevDaxProviderProxyPoolTest, umfIpcTest,
```

Comment:
I think this instantiation of the devdax + jemalloc should be done in the `provider_devdax_memory.cpp`. Something like this:

```c++
static std::vector<ipcTestParams> ipcProxyPoolTestParamsList = {
    {umfProxyPoolOps(), nullptr, umfDevDaxMemoryProviderOps(),
    &defaultDevDaxParams, &hostAccessor, false},
#ifdef UMF_JEMALLOC_ENABLED // I do not remember exact macros
    {umfJemallocPoolOps(), nullptr, umfDevDaxMemoryProviderOps(),
     &defaultDevDaxParams, &hostAccessor, false},
#endif
#ifdef UMF_SCALABLE_POOL_ENABLED // I do not remember exact macros
    {umfScalablePoolOps(), nullptr, umfDevDaxMemoryProviderOps(),
     &defaultDevDaxParams, &hostAccessor, false},
#endif
```

I think it is natural to instantiate tests for a particular provider with all available pools in a single palce

Reply:
you can refer to the #860. How I enabled IPC tests for OS provider with scalable pool